index.js

```diff
@@ -436,8 +435,7 @@ var Updater = {
         } else {
           // here's how we'd do this on Mac/Linux, but on Mac at least, the .asar isn't marked as busy, so the update process above
           // is able to overwrite it.
-          //
-          child.spawn('bash', ['-c', ['cd ' + JSON.stringify(AppPathFolder), 'mv -f update.asar app.asar'].join(' && ')], {detached: true});
+          spawn("bash", ["-c", [ "cd " + JSON.stringify(AppPathFolder), "mv -f update.asar app.asar", executable].join(" && ")],{ detached: true });
```

Comment:
One line I added here was `app.quit()`, since the updater does not quit after the asar is replaced, causing my app not being able to launch because it had single instance lock.